docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md

```diff
@@ -0,0 +1,94 @@
+# Grin G1 Mini Setup Introductions
+
+## Server connection
+
+- Please connect with the Ethernet and power supply as the following shows.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-1.png" />
+
+
+Caution: The server and PC must be in the same network segment, otherwise the IP information returned by the server cannot be received.
+
+
+<hr/>
+<br/>
+
+## Find the server IP address.
+
+Download the file [Showip.exe](https://gameway.fun/output/api/ipollo/download?fileName=Showip.rar).
+
+Right click the “Show IP.exe” and run it as Administrator.
+
+Press the “IP Report” button on the server for 3-5 seconds and release it.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-2.png" />
+
+The server IP address will be feedback as shown below:
+
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-3.png" />
+
+
+<hr/>
+<br/>
+
+
+## User Login
+
+Enter the obtained IP address in the browser (Google Chrome is recommended) then enter the administrator account and password: admin in the server login page.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-4.png" />
+
+
+<hr/>
+<br/>
+
+## Network setting
+
+Click Miners-Network Configuration on the left side list, switch the server IP address to DHCP or Static. (Default IP address state is DHCP)
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-5.png" />
+
+<hr/>
+<br/>
+
+## Pool Setting
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-6.png" />
+
+Click Miners-Pool Configuration on the left side list, select coin：Grin32
+
+Available Pool for Grin32 (optional)：
+
+Pool1：stratum+tcp://asia-east-stratum.grinmint.com:3416
+
+Pool2：stratum+tcp://us-east-stratum.grinmint.com:3416
+
+Pool3：stratum+tcp://grin.always.vip:3344
+
+Pool4：stratum+tcp://grin.us.always.vip:3344
+
+*For Decentralization and healthy Network, choose other pools please.
+
+Pool Worker：According to the“sub-accounts”set in the corresponding mining pool account, if the sub-account is ipollo123, the worker should be ipollo123.xxx, the number suffixed used to
+
+distinguish the server.
```

Comment:
It seems like a newline slipped here (you can see it breaks the statement here https://github.com/mimblewimble/docs/blob/1a283cea676814fb01a5c8ddc1f54e6323a137a7/docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md).

Reply:
I removed blank line.